### PR TITLE
separeted icon from text dom element; simplified selectors;

### PR DESCRIPTION
**Pull request by rfreitas**
### Problem

Fix to https://github.com/onfido/onfido-sdk-ui/pull/186

Separated the icon and title text into their own dom elements.
Reduced the number of selectors on the html side.

### CODE REVIEW

**Comment by rfreitas** [comment]
@stefaniacardenas might be interesting for you from an information point of view.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://189-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-189-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Error/index.js [hunk 1]
```diff
@@ -7,16 +7,12 @@ import style from './style.css'
```
Comment (hjheath) [inline src/components/Error/index.js]: you can remove the classNames import

File: src/components/Error/style.css [hunk 2]
```diff
@@ -35,30 +37,54 @@
...
+
+.title-text {
   display: inline-block;
   font-size: 24px;
   font-weight: 600;
-  line-height: 28px;
-  padding-left: 40px;
-  margin-bottom: 4px;
+  line-height: @title-height;
+  padding-left: 16px + @icon-width;
```
Comment (hjheath) [inline src/components/Error/style.css]: I really like doing this maths explicitly

**Comment by rfreitas** [comment]
Travis automatic deployment: https://189-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**